backend/tests/bulkAppointmentsFormRedirect.test.js

```diff
@@ -6,25 +6,11 @@ const mockRes = { redirect: jest.fn() }
 describe('add appointment redirect', () => {
```

Comment:
Is this part of the new bulk appointment work?

Reply:
It was put together to cater for the session clear and redirect.  I removed the session part but we've no need for any of it.  About to remove the whole thing.